apps/spruce/src/pages/adminSettings/tabs/GeneralTab/OtherTab/schemaFields.ts

```diff
@@ -0,0 +1,546 @@
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import widgets from "components/SpruceForm/Widgets";
+import {
+  arrayItemCSS,
+  objectGridCss,
+  fullWidthCss,
+  nestedObjectGridCss,
+} from "../../sharedStyles";
+
+export const miscSettings = {
+  schema: {
+    configDir: {
+      type: "string" as const,
+      title: "Config Directory",
+    },
+    domainName: {
+      type: "string" as const,
+      title: "Domain Name",
+    },
+    githubPRCreatorOrg: {
+      type: "string" as const,
+      title: "GitHub PR Creator Organization",
+    },
+    shutdownWaitSeconds: {
+      type: "number" as const,
+      title: "Shutdown Wait Time (secs)",
+    },
+    githubWebhookSecret: {
+      type: "string" as const,
+      title: "GitHub Webhook Secret",
+    },
+    pprofPort: {
+      type: "string" as const,
+      title: "PProf Port",
+    },
+    logPath: {
+      type: "string" as const,
+      title: "Log Path",
+    },
+
+    perfMonitoringKanopyURL: {
+      type: "string" as const,
+      title: "Performance Monitoring Kanopy URL",
+      format: "validURL",
+    },
+    perfMonitoringURL: {
+      type: "string" as const,
+      title: "Performance Monitoring URL",
+      format: "validURL",
+    },
+    githubOrgs: {
+      type: "array" as const,
+      title: "GitHub Organizations",
+      items: {
+        type: "string" as const,
+      },
+    },
+    releaseMode: {
+      type: "object" as const,
+      title: "Release Mode",
+      properties: {
+        distroMaxHostsFactor: {
+          type: "number" as const,
+          title: "Distro Max Hosts Factor",
+        },
+        targetTimeSecondsOverride: {
+          type: "number" as const,
+          title: "Target Time Override (secs)",
+        },
+        idleTimeSecondsOverride: {
+          type: "number" as const,
+          title: "Idle Time Override (secs)",
+        },
+      },
+    },
+  },
```

Comment:
Could we add descriptions to these (the same ones on the current Admin Settings page)?